tests/vmi_configuration_test.go

```diff
@@ -235,10 +235,10 @@ var _ = Describe("[Serial]Configurations", func() {
 			It("[test_id:4624]should set a correct memory units", func() {
 				vmi.Spec.Domain.Resources = v1.ResourceRequirements{
 					Requests: kubev1.ResourceList{
-						kubev1.ResourceMemory: resource.MustParse("512Mi"),
+						kubev1.ResourceMemory: resource.MustParse("64Mi"),
```

Comment:
How is this change related here? (Same on the next line)

Reply:
It reduces the memory footprint of the vmi configuration tests where not so much memory is needed, so that it can be run in parallel.

Reply:
Ok, please consider to add this info to the commit. When walking the code, I use the history to answer the "why" part when possible and having this info on the commit may help.

Reply:
Added.